### ISSUE TITLE
Feature request: Show in App Grid

### ISSUE THREAD

**Issue by PhilDevProg** [nlpsuge/Always-Show-Titles-In-Overview#16]
Having the option to apply these features also to the window previews in the App Grid would make the extension even better for me since I use the App Grid a lot.
Btw, thank you for making this extension.

**Comment by nlpsuge** [comment]
> the window previews in the App Grid

Did you mean the window previews in those workspace thumbnails box under the search entry? If so, I think those window previews are too small to put something like icon or title on them...

**Comment by PhilDevProg** [comment]
> > the window previews in the App Grid
> 
> Did you mean the window previews in those workspace thumbnails box under the search entry? If so, I think those window previews are too small to put something like icon or title on them... 

Yes, these are what I mean. A title might be too much but the App icon (maybe with a different scale factor than the one in the overview) or/and the close button should be usable (and would improve it a lot in my opinion) if you only have one or two windows on one virtual desktop.
